php/class-coauthors-plus.php

```diff
@@ -107,6 +107,18 @@ public function __construct() {
 
 		// REST API: Depending on user capabilities, hide author term description.
 		add_action( 'rest_prepare_author', array( $this, 'conditionally_hide_author_term_description' ) );
+
+		// Add Co-Author select field to the Bulk Edit actions form.
+		add_action( 'bulk_edit_custom_box', array( $this, '_action_bulk_edit_custom_box' ), 10, 2 );
+
+		// Update Co-Authors when bulk editing posts.
+		// bulk_edit_posts was introduced in WordPress 6.3, so we need to check the version.
+		global $wp_version;
+		if ( version_compare( $wp_version, '6.3', '>=' ) ) {
+			add_action( 'bulk_edit_posts', array( $this, 'action_bulk_edit_update_coauthors' ), 10, 2 );
+		} else {
+			add_action( 'wp_insert_post_data', array( $this, 'action_bulk_edit_update_coauthors' ), 10, 2 );
```

Comment:
I don't think we need or should keep backward compatibility with WP < 6.3... imo it's better if we limit this to WP > 6.3

Reply:
Thanks! I don't think we need to bum the `Requires at least` flag to 6.3. It's not that the whole plugin requires it. Instead we can just do a check for the version to decide whether to enable this particular feature or not.